lib/computer_player.rb

```diff
@@ -0,0 +1,19 @@
+class ComputerPlayer
```

Comment:
I like the symmetry/polymorphism you created with a computer player and a player!